subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaDocOptions.java

```diff
@@ -161,15 +163,22 @@ public void setBottom(String bottom) {
 
     /**
      * Returns the style sheet to override default style.
+     *
+     * @deprecated Scaladoc does not support to set a stylesheet any more (Scala 2.11).
      */
+    @PathSensitive(PathSensitivity.ABSOLUTE)
     @InputFile @Optional
+    @Deprecated
     public File getStyleSheet() {
         return styleSheet;
     }
 
     /**
      * Sets the style sheet to override default style.
+     *
+     * @deprecated Scaladoc does not support to set a stylesheet any more (Scala 2.11).
      */
+    @Deprecated
     public void setStyleSheet(File styleSheet) {
```

Comment:
To add to that, it seems that the property was called `stylesheetfile` - so I guessed it never worked. If someone complains and can explain us how that freaking flag is supposed to work I am happy to un-deprecate the option again. But only if we add an integration test.